packages/docusaurus-theme-classic/src/utils/index.ts

```diff
@@ -8,5 +8,5 @@
 // Compare the 2 paths, ignoring trailing /
 export const isSamePath = (path1, path2) => {
   const normalize = (str) => (str.endsWith('/') ? str : `${str}/`);
-  return normalize(path1) === normalize(path2);
+  return path1 && path2 ? normalize(path1) === normalize(path2) : true;
```

Comment:
thanks

What if one path is undefined and the other is defined, wouldn't it wrongly return true?

Reply:
This function only makes sense if a path exists as a field, otherwise it doesn't matter what it returns. If path is missing this function shouldn't be called at all, which happens when Navbar has a drop-down which holds a list of entries instead of a path. 

With this quick fix I made it check for undefined and return arbitrary value further to hope the underlying logic works and with "true" it does work, probably with false as well as there's just no path.

The real fix should be somewhere in the navbar generation and there's obviously no test for a navbar with a drop-down which would catch this. 

In 2.0.0-alpha.65 it still works fine. With .66 it breaks. Removing a drop-down from docusaurus config makes it work with with .66 as well.

The fix I propose is doing nothing bad, it simply does the check only if none of the values are undefined and returns smth to avoid 'undefined' further.

Reply:
It's more of a workaround. I'll see if I can fix it in the reverting of the Navbar